docs/source/api/aihwkit.utils.rst

```diff
@@ -13,4 +13,6 @@ Submodules
    :maxdepth: 4
 
    aihwkit.utils.analog_info
+   aihwkit.utils.fitting
    aihwkit.utils.visualization
+   aihwkit.utils.visualization_web
```

Comment:
`visualization_web` not needed

Reply:
I added this to the excluded path in the Makefile.